src/components/sections/projects/tabs/ProjectRetrospection.tsx

```
@@ -0,0 +1,71 @@
+import { Card, CardContent, CardHeader, CardTitle } from '@/components/ui/card';
+import { TabsContent } from '@/components/ui/tabs';
+import type { ProjectRetrospection } from '@/constants/projects';
+import { Lightbulb, ThumbsDown, ThumbsUp } from 'lucide-react';
+
+interface ProjectRetrospectionProps {
+  retrospection: ProjectRetrospection[];
+}
+
+const ProjectRetrospection = ({ retrospection }: ProjectRetrospectionProps) => {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'ProjectRetrospection'. Consider to delete it or rename it.

'ProjectRetrospection' is defined here:


(lint/suspicious/noRedeclare)